Description: debianization -- FireQOS
 Make /etc/fireqos the configuration folder for FireQOS;
 see debian/fireqos.README.Debian for details.
Author: Jerome Benoit <calculus@rezozer.net>
Last-Update: 2016-11-22

--- a/sbin/fireqos.in
+++ b/sbin/fireqos.in
@@ -49,7 +49,7 @@
 	fi
 done
 
-FIREHOL_CONFIG_DIR="$sysconfdir_POST/firehol"
+FIREHOL_CONFIG_DIR="$sysconfdir_POST/fireqos"
 common_disable_localization || exit
 common_public_umask || exit
 common_require_root || exit
--- a/doc/fireqos/fireqos.1.md
+++ b/doc/fireqos/fireqos.1.md
@@ -25,7 +25,7 @@
 Run without any arguments, `fireqos` will present some help on usage.
 
 When given *CONFIGFILE*, `fireqos` will use the named file instead of
-`/etc/firehol/fireqos.conf` as its configuration.
+`/etc/fireqos/fireqos.conf` as its configuration.
 
 The parameter *name* always refers to an interface name from the
 configuration file. The parameter *class* always refers to a named class
@@ -96,7 +96,7 @@
 
 # FILES
 
-`/etc/firehol/fireqos.conf`
+`/etc/fireqos/fireqos.conf`
 
 # SEE ALSO
 
--- a/doc/fireqos/fireqos-conf.5.md
+++ b/doc/fireqos/fireqos-conf.5.md
@@ -15,7 +15,7 @@
 This file defines the traffic shaping that will be applied by
 [fireqos(1)][].
 
-The default configuration file is `/etc/firehol/fireqos.conf`. It can be
+The default configuration file is `/etc/fireqos/fireqos.conf`. It can be
 overridden from the command line.
 
 A configuration consists of a number of input and output `interface`
